apis/bigquerybiglake/v1alpha1/table_reference.go

```diff
@@ -16,68 +16,55 @@ package v1alpha1
 
 import (
 	"context"
-	"fmt"
 
 	refsv1beta1 "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
-	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/k8s"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var _ refsv1beta1.ExternalNormalizer = &TableRef{}
```

Comment:
This aligns with the Apigee code format. But we also have [a discussion](https://github.com/GoogleCloudPlatform/k8s-config-connector/commit/9d01d994f2fe3c9ea0f07c59b863fe565d2f8322#diff-39e83069bbca639a3e846abda2ef4b103bc0360dce3183652d3b606de8bf1111) about not calling a generic function to normalize reference (more like a magic machine). cc @justinsb

Reply:
The link to the discussion is broken for me.  But I think we do want a consistent normalizer, rather than repeating the code every time.